src/components/ModalWindow/ModalWindow.scss

```diff
@@ -0,0 +1,26 @@
+@use '../../styles/typography' as *;
+@use '../../styles/variables' as *;
+.modal-background{
+    position: fixed;
+    width: 100vw;
+    height: 100vh;
+    background-color: hwb(0 0% 100% / 0.20);
+}
+
+.modal{
+    position: fixed;
+    width: 774px;
+    height: 735px;
+    left: 477px;
+    top: 191px;
+    padding: 20px;
```

Comment:
was there a reason pixels were used here?

Reply:
this has now been changed to rem